script/Ciel/Day 7/07_01_CIEL07_1_2.txt

```diff
@@ -1,53 +1,54 @@
 [sha:5f3ad6ee0410701b739a79eaee295a1a3942be3a]{
 -- Page 10004, Offset 15010.
 -- 購買で揚げパンを買おう。
-// 揚げパン is fried bread but it sounds weird so I've left as just bread
-I'll buy some bread at the school store.
+I'll buy some fried buns at the school store.
```

Comment:
This was actually the line that made be go and change everything, because the next line is
> I think I could probably put away four of them given how I'm feeling today.

and "I think I could probably put away four [bread]" doesn't make any sense. Is it better if I drop the 'fried' again?